bc_obps/compliance/service/automated_process/compliance_handlers.py

```diff
@@ -83,6 +89,15 @@ def handle(self, invoice: ElicensingInvoice) -> None:
             )
             logger.info(f"Updated penalty status to ACCRUING for obligation {obligation.obligation_id}")
 
+        # Set both penalties for this obligation to ACCRUING when the obligation invoice is overdue
+        CompliancePenalty.objects.filter(
+            compliance_obligation=obligation,
+            penalty_type__in=[
+                CompliancePenalty.PenaltyType.AUTOMATIC_OVERDUE,
+                CompliancePenalty.PenaltyType.LATE_SUBMISSION,
+            ],
+        ).update(status=CompliancePenalty.Status.ACCRUING)
+
```

Comment:
CompliancePenalty records will not exist while the penalty is accruing. They are created when the obligation is paid in full